Sources/SwiftKafka/KafkaProducer.swift

```diff
@@ -0,0 +1,31 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-gsoc open source project
+//
+// Copyright (c) 2022 Apple Inc. and the swift-kafka-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+import Logging
+
+public struct KafkaProducer {
```

Comment:
Let's add some doc comments here

Reply:
We will see but I think this will be a `final class` in the long run

Reply:
> Let's add some doc comments here

Implemented in PR on top of this one https://github.com/felixschlegel/swift-kafka-gsoc/pull/1